core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -653,6 +653,19 @@ class ZkUtils(val zkClient: ZkClient,
     (dataStr, stat)
   }
 
+  def readData(path: String, returnNoneIfPathNotExists: Boolean): Option[String] = {
```

Comment:
My bad, I didn't realise that we already had a `readData` method. Yes, what you suggest seems fine.